Doc/c-api/veryhigh.rst

```diff
@@ -301,7 +301,7 @@ the same library that the Python runtime is using.
    set to *NULL*.
 
 
-.. c:function:: PyObject* PyEval_EvalCodeEx(PyObject *co, PyObject *globals, PyObject *locals, PyObject **args, int argcount, PyObject **kws, int kwcount, PyObject **defs, int defcount, PyObject *kwdefs, PyObject *closure)
+.. c:function:: PyObject* PyEval_EvalCodeEx(PyObject *co, PyObject *globals, PyObject *locals, PyObject * const *args, int argcount, PyObject * const *kws, int kwcount, PyObject * const *defs, int defcount, PyObject *kwdefs, PyObject *closure)
```

Comment:
A nit, but the spacing in this PR looks weird to me. How about `PyObject *const *kws`?

Reply:
Done.